ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-release-4.20.yaml

```diff
@@ -375,7 +375,7 @@ tests:
       ITERATION_MULTIPLIER_ENV: "15"
       OUTPUT_FORMAT: JUNIT
       RUN_ORION: "true"
-      VERSION: "4.19"
+      VERSION: "4.20"
```

Comment:
FWIW there are other test jobs that use `oc get clusterversion` in the test scripts to figure out the version